src/main/java/com/limechain/transaction/dto/ApplyExtrinsicResult.java

```diff
@@ -0,0 +1,14 @@
+package com.limechain.transaction.dto;
+
+import lombok.Data;
+import org.jetbrains.annotations.Nullable;
+
+@Data
+public class ApplyExtrinsicResult {
+
+    @Nullable
+    DispatchOutcome outcome;
```

Comment:
Initially I wanted to use lombok's `@Value`. It seems I forgot to add back the private modifiers. The `DispatchOutcome` is alright since we want to have its fields `private final`, hence the use of `@Value`.